### ISSUE TITLE
Nice UI

### ISSUE THREAD

**Issue by electricmonk** [electricmonk/ts-react-tdd#8]

**Comment by silicakes** [comment]
I suggest using a UI component library (MUI v5 is a favorite of mine), without writing any CSS at all.
We can pick up a theme we like and only deal with the functionality. Thoughts?

**Comment by electricmonk** [comment]
I agree. Not sure about MUI specifically. The prime KPI from my perspective is testability. For instance, antd is hell to test.
